app/webpacker/css/admin/products_v3.scss

```diff
@@ -82,6 +82,7 @@
     font-size: $h6-size;
     font-weight: 600;
     gap: 20px;
+    margin-top: 116px;
```

Comment:
I'm wondering if this message gets pushed below the fold on a smaller screen (like mobile or tablet), are you able to check?
This might be a good case for the `vh` unit.

Reply:
On second thought, I don't think this is worth bothering with. We don't want to support screens that small right now anyway.